tests/test_query.py

```diff
@@ -42,7 +42,13 @@ def mock_successful_get(test_data):
     """Mock get function that returns non-empty query results."""
 
     async def mock_ret(
```

Comment:
Was this supposed to be called `..._get`?

Reply:
I think so but I'll check with Alyssa to be sure.

Reply:
I think renaming it to mock_return will clear things up